src/components/ui/Merge/Goodle/GoodleMergeForm.tsx

```
@@ -0,0 +1,218 @@
+          </div>
+        </div>
+
+        <div className="flex flex-col gap-3 w-full sm:w-auto ">
+          <div className="flex-1 flex flex-col sm:flex-row items-center gap-3 ">
+            <Button
+              className="flex items-center justify-center gap-3 w-full md:w-auto md:h-[62px]"
+              disabled={!address || !amount}
+              onClick={
+                isConnected
+                  ? () => handleMerge()
+                  : () => openConnectModal && openConnectModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)